dockstore-common/src/main/java/io/dockstore/common/model/DOIMessage.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *    Copyright 2017 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package io.dockstore.common.model;
+
+public class DOIMessage extends BasicMessage {
+    private String targetEntry;
```

Comment:
What is the targetEntry?

Reply:
The entry (tool or workflow) that should have a DOI requested for it. 
I'll add some Javadoc